Source/Preprocessor/cpp.c

```diff
@@ -1009,7 +1010,7 @@ static String *expand_macro(String *name, List *args, String *line_file) {
   /* Expand this macro even further */
   Setattr(macro, kpp_expanded, "1");
 
-  e = Preprocessor_replace(ns);
+  e = Preprocessor_replace(ns, line_file);
```

Comment:
I'm confused by this part, and wonder if you meant the new parameter passed here to be `next`?

The new second parameter to `Preprocessor_replace()` is called `next` and otherwise the new `next` parameter to `expand_macro()` doesn't seem to get used anywhere...